pkgs/development/compilers/llvm/git/default.nix

```
@@ -17,15 +17,20 @@
     else pkgs.bintools
 , darwin
 # LLVM release information; specify one of these but not both:
-, gitRelease ? null
+, gitRelease ? {
+    version = "19.0.0-git";
+    rev = "65058a8d732c3c41664a4dad1a1ae2a504d5c98e";
+    rev-version = "unstable-2024-03-16";
```

```suggestion
    rev-version = "19.0.0-unstable-2024-03-16";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming

---

Fixed